packages/desktop/features/network.features.ts

```diff
@@ -6,7 +6,7 @@ const networkFeatures: INetworkFeatures = {
         enabled: true,
     },
     config: {
-        enabled: false,
+        enabled: true,
```

Comment:
is it now time to enabled it per default?

Reply:
Yup, let's do it! We have a separate release branch for the SH fix anyway.